### DISCUSSION TITLE
Galaxy Brain

### DISCUSSION THREAD

**Discussion by thatwonguy** [Schweinepriester/github-profile-achievements#915] [Q&A]
Answer: To recap regarding the galaxy brain: 
![image](https://github.com/Schweinepriester/github-profile-achievements/assets/78534460/d2605614-345d-4f83-81d0-15e4e7be7567)

To unlock the Galaxy Brain badge on GitHub, you need to answer at least 2 discussions. The discussions must be in a public repository, and you cannot mark your own answers to your own questions.

**Comment by Saqib198** [comment]
Thanks For Recap...

**Comment by 17351269** [comment]
Yes

در تاریخ جمعه 24 مه 2024، 22:44 Muhammad Saqib Javed <
***@***.***> نوشت:

**Comment by 17351269** [comment]
Tanks

در تاریخ شنبه 25 مه 2024، 1:35 Vahid Bakhshi <
***@***.***> نوشت:

**Comment by LaFilleDuCode** [comment]
merci

**Comment by JBlitzar** [comment]
That's right. I would strongly advise you to mark this as the correct answer so I can get galaxy brain
